packages/edit-site/src/components/list/index.js

```diff
@@ -73,23 +77,38 @@ export default function List() {
 		  }
 		: undefined;
 
+	function onPluginAreaError( name ) {
+		createErrorNotice(
+			sprintf(
+				/* translators: %s: plugin name */
+				__(
+					'The "%s" plugin has encountered an error and cannot be rendered.'
+				),
+				name
+			)
+		);
+	}
+
 	return (
-		<InterfaceSkeleton
-			className={ classnames( 'edit-site-list', {
-				'is-navigation-open': isNavigationOpen,
-			} ) }
-			labels={ {
-				drawer: __( 'Navigation Sidebar' ),
-				...detailedRegionLabels,
-			} }
-			header={ <Header templateType={ templateType } /> }
-			drawer={ <NavigationSidebar.Slot /> }
-			notices={ <EditorSnackbars /> }
-			content={ <Table templateType={ templateType } /> }
-			shortcuts={ {
-				previous: previousShortcut,
-				next: nextShortcut,
-			} }
-		/>
+		<>
+			<InterfaceSkeleton
+				className={ classnames( 'edit-site-list', {
+					'is-navigation-open': isNavigationOpen,
+				} ) }
+				labels={ {
+					drawer: __( 'Navigation Sidebar' ),
+					...detailedRegionLabels,
+				} }
+				header={ <Header templateType={ templateType } /> }
+				drawer={ <NavigationSidebar.Slot /> }
+				notices={ <EditorSnackbars /> }
+				content={ <Table templateType={ templateType } /> }
+				shortcuts={ {
+					previous: previousShortcut,
+					next: nextShortcut,
+				} }
+			/>
+			<PluginArea onError={ onPluginAreaError } />
```

Comment:
Another possibility is to lift this element from both `<List />` and `<Editor />` components and put it on `EditSiteApp` component. Let me know which one you prefer. Both are fine for me.

Reply:
@kevin940726, any preferences on this?

Reply:
A common `<App>` parent component sounds good, I've already done the same thing in #37314, so I don't mind merging this as is and we can port it in that PR later.

Reply:
For now, I'll lift them both to `<EditSiteApp />`, the same place we have the `<NavigationSidebar />` element. Then, in your PR, you can refactor it to the `<App />` component you've created 👍